frontend/src/components/WofInfoBox.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+    <wof-popper> <!--problem with WofPopper - doesnt work-->
+    <table :class="`WofBox WofBox--${this.type}`">
+        <tr :id="`first-row`">
+            <th :id="`title`">{{ title }}</th>
+            <th :id="`close-window`">
+                <wof-button :type="this.type" :id="`close-window-button`" @click="$emit('close')">
+                <wof-icon icon="wof-cancel" size=2  />
+                </wof-button>
+            </th>
+        </tr>
+        <tr>
+        <th colspan="2" id="content">{{ content }}</th>
+        </tr>
+    </table>
+    </wof-popper>
+</template>
+
+<script>
+import WofButton from './WofButton.vue'
+import WofPopper from './WofPopper.vue'//problem with WofPopper - doesnt work
+
+export default {
+  components: { WofButton, WofPopper },//problem with WofPopper - doesnt work
+    name: "WofBox",
+
+    props: {
+        title: {
+            type: String,
+            default: "Title",
+        },
+        content: {
+            type: String,
+            default: "Content of box",
+        },
+        type: {
+            type: String,
+            required: true,
+            default: "info",
+            validator(value){
+                return["error", "warning", "success", "info"].includes(value)
+            }
+        }  
+    }
+}
+</script>
+
+
+<style lang="less">
+@import './common.less';
+
+.WofBox{
+    background-color: @secondary-color;
+    width: 500px;
+    font-size: 24px;
+    word-wrap: break-word;
+    color: white;
+    top:50%;
+    left:50%;
+    position:fixed;
+    -webkit-transform: translate(-50%, -50%);
+    transform: translate(-50%, -50%);
```

Comment:
WofInfoBox should display in center of his parent element, not in center of entire site. 
We use popper to display it in center of a site.